src/libguac/Makefile.am

```diff
@@ -171,5 +171,6 @@ libguac_la_LDFLAGS =     \
     @UUID_LIBS@          \
     @VORBIS_LIBS@        \
     @WEBP_LIBS@          \
-    @WINSOCK_LIBS@
+    @WINSOCK_LIBS@       \
+    @RT_LIBS@
```

Comment:
Please maintain alphabetical ordering.

Reply:
OK, I've changed code according to your opinion.